lib/vagrant/cli.rb

```diff
@@ -30,6 +31,20 @@ def execute
         return 0
       end
 
+      if @main_args.include?("-install-autocomplete")
+        # Install autocomplete scripts
+        written_paths = Vagrant::Util::InstallCLIAutocomplete.install
+        if written_paths.length > 0
```

Comment:
Hmm, it seems like it's possible written_paths could return nil? I think this line would fail if so with no method `length` for nil class.